README.md

```diff
@@ -1,7 +1,11 @@
 ## Configure
 ```
+### From source code
 sudo python3 -m pip install pipenv 
-alias standup-notes=<workspace>/standup_notes/bin/standup-notes
+python setup.py install
```

Comment:
This command didn't work for me because I needed:
1. To call `python3`; and
2. To use `sudo`.

```
sudo python3 setup.py install
```
So, went to the internet to find some information on _"Should I call `python` or `python3` if I want to run Python 3.x?_ and I found [PEP 394 -- The "python" Command on Unix-Like Systems](https://www.python.org/dev/peps/pep-0394/). If you have some time, give it a quick read. It's pretty interesting to see how much thought has gone into _exactly_ this scenario. 

Here's what I took away (and I could easily be wrong - it's long lol): 
- If we know we want to run Python 3.x, we should call `python3` directly. 
- However, (I think this PEP describes how) this _isn't_ necessary if you've created a virtual environment where you know the only interpreter available _is_ Python 3.x.
----
TLDR; I think this should be, at a minimum:

```
python3 setup.py install
```
and then caveat it (in a comment) with something like
> \# This might need to be called using sudo on *nix systems.

Reply:
Cool, I'll update that to include sudo. I have my python 3 installation in a location that doesn't require sudo to run, but won't hurt to add it in.